src/main/java/com/codeborne/selenide/drivercommands/Navigator.java

```diff
@@ -20,9 +20,9 @@
 import static java.util.regex.Pattern.DOTALL;
 
 public class Navigator {
-  protected Pattern absoluteUrlRegex = Pattern.compile("^[a-zA-Z]+:.*", DOTALL);
+  protected static final Pattern ABSOLUTE_URL_REGEX = Pattern.compile("^[a-zA-Z]+:.*", DOTALL);
```

Comment:
My initial idea was to make all such fields protected and mutable to allow users to customize Selenide behaviour or fix some bugs on the fly.

But well, I guess nobody actually uses it, so let's make them final.
But I would also make it private then.